depmapomics/config_dev.py

```diff
@@ -393,6 +428,25 @@
     "CALL": "Status",
 }
 
+PURECN_COLRENAMING = {
```

Comment:
I would have guessed config_*.py would contain those variables that are different between the different environments. This to me sounds less like "config" and more like a constant that you want to use. (Different environments aren't going to name the columns differently, are they?)

As this is set up, this will need to be duplicated in config_prod.py, correct?

Reply:
Yes that's a good point. Currently config_prod and config_dev have a lot of overlaps. Maybe we'd want another script called param or something to store the shared constants?